### ISSUE TITLE
Python arcade isn't working under WSL, however pyglet does work so it must be possible

### ISSUE THREAD

**Issue by originalsurfmex** [pvcraven/arcade#465]
### Actual behavior:
```
~$ python3 -m arcade.examples.snow
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/user/.local/lib/python3.6/site-packages/arcade/examples/snow.py", line 113, in <module>
    main()
  File "/home/user/.local/lib/python3.6/site-packages/arcade/examples/snow.py", line 108, in main
    window = MyGame(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
  File "/home/user/.local/lib/python3.6/site-packages/arcade/examples/snow.py", line 45, in __init__
    super().__init__(width, height, title)
  File "/home/user/.local/lib/python3.6/site-packages/arcade/application.py", line 51, in __init__
    resizable=resizable, config=config)
  File "/home/user/.local/lib/python3.6/site-packages/pyglet/window/xlib/__init__.py", line 171, in __init__
    super(XlibWindow, self).__init__(*args, **kwargs)
  File "/home/user/.local/lib/python3.6/site-packages/pyglet/window/__init__.py", line 593, in __init__
    config = screen.get_best_config(config)
  File "/home/user/.local/lib/python3.6/site-packages/pyglet/canvas/base.py", line 197, in get_best_config
    raise window.NoSuchConfigException()
pyglet.window.NoSuchConfigException
```

### Expected behavior:
For the built-in snow example to run.

### Steps to reproduce/example code:
Enable WSL on windows, install a linux (I used Ubuntu from Windows Store)
Install python arcade and all dependencies via pip3 and/or package manager
Install VcXsrv and enable the dbus, OpenGL and LIBGL fix's if needed.
Open a terminal and run this as a test.  If it works then arcade should also work:
```
wget https://bitbucket.org/pyglet/pyglet/downloads/pyglet-1.3.0.zip
tar -xzvf pyglet-1.3.0.zip
cd pyglet-1.3.0
python examples/graphics.py
```
If that worked then open a terminal and run this:
```
python3 -m arcade.examples.snow
```

## Enhancement request:
Help identify why arcade isn't working and either instruct us how to run it or update it's files to run under WSL

### What should be added/changed?
Not sure.

### What would it help with?
WSL is a growing environment and works well for Linux and OpenGL.  Tools like emacs that work exceptionally well on Linux but only pretty good on Windows could now be used.

## Documentation request:
Instructions to get it running on WSL, if needed.

**Comment by pvcraven** [comment]
You need OpenGL 3.3+ for it to work. 

I think you can find your version with: 

`glxinfo | grep "OpenGL version"`

Pyglet (right now) doesn't require that high a version of OpenGl, which is why it works.

**Comment by originalsurfmex** [comment]
Gotcha, thanks very much for explaining.  I did some digging around and it seems that no xserver for windows allows fully accelerated opengl.  Is there a way for me to lower the requirements on arcade by editing some of the files?

**Comment by originalsurfmex** [comment]
Ok I did a little research on this.  It appears even getting OpenGL 3.3 to work isn't really possible with the current sofware rendering on X servers.

I tried a couple of hacking things from here, and while they speeded up glxgears greatly, they couldn't fool arcade.  


https://askubuntu.com/questions/850900/why-is-my-opengl-version-stuck-at-3-0-despite-new-hardware-software

Thanks for the help, looks like WSL isn't there yet.
